### PR TITLE
Jcli encrypt vote tools

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Added 3 commands to jcli to generate vote committee related keys.

### CODE REVIEW

File: jcli/src/jcli_app/vote/committee_communication_key.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct CommitteeCommunicationKey {}
+
+impl CommitteeCommunicationKey {
+    pub fn exec(&self) -> Result<(), super::Error> {
+        let key = generate_committee_member_keys();
+        println!("Private: {}", hex::encode(key.to_bytes()));
+        println!("Public: {}", hex::encode(key.to_public().to_bytes()));
```
Comment (danielSanchezQ) [inline jcli/src/jcli_app/vote/committee_communication_key.rs:18]: I went for the shortest path here. But I'm not sure if what is the standard we use for outputting data. Will check it too.

File: jcli/src/jcli_app/vote/committee_member_key.rs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    #[test]
+    fn test_keys_dance() {
+        let mut rng = ChaCha20Rng::from_seed([0u8; 32]);
+        let mck = MemberCommunicationKey::new(&mut rng);
+        let pk_bytes = mck.to_public().to_bytes();
+        let pk = PublicKey::from_bytes(&pk_bytes).unwrap();
+        MemberCommunicationPublicKey::from_public_key(pk);
+    }
+}
```
Comment (danielSanchezQ) [inline jcli/src/jcli_app/vote/committee_member_key.rs:68]: Ups, this should be removed...

File: jcli/src/jcli_app/mod.rs [hunk 3]
```diff
@@ -52,6 +53,8 @@ pub enum JCliCommand {
     AutoCompletion(auto_completion::AutoCompletion),
     /// Utilities that perform specialized tasks
     Utils(utils::Utils),
+    /// Vote related operations
+    Votes(vote::Vote),
```
Comment (enzoc4) [inline jcli/src/jcli_app/mod.rs:57]: why not put the key generation under `Key`?

Comment (danielSanchezQ) [reply jcli/src/jcli_app/mod.rs:57]: Actually yes, I didn't even think about it, I just wanted to keep all voting-related tools together. But I'm down for the key option too.

**Comment by mzabaluev** [comment]
Superseded by #2603.

**Pull request closed without merging**